partner_firstname/__openerp__.py

```diff
@@ -15,35 +17,34 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
 {
-    'name': 'Partner first name, last name',
+    'name': 'Partner first name and last name',
     'description': """
-This module splits first name and last name for non company partners
-====================================================================
+Split first name and last name for non company partners
+=======================================================
 
-The field 'name' becomes a stored function field concatenating lastname and
-firstname
-Note: in version 7.0, installing this module invalidates a yaml test in the
-'edi' module
+The field *name* becomes a stored function field concatenating the last name
+and the first name.
 
 Contributors
-============
-Jonathan Nemry <jonathan.nemry@acsone.eu>
-Olivier Laurent <olivier.laurent@acsone.eu>
+~~~~~~~~~~~~
+
+- Jonathan Nemry <jonathan.nemry@acsone.eu>
```

Comment:
Can you also add Nicolas Bessi on top of contributors? He is the autor of the module so he deserve to be in the readme file